buildpacks/add-cnb-require-jdk/buildpack.toml

```diff
@@ -0,0 +1,19 @@
+api = "0.9"
+
+[buildpack]
+id = "heroku/add-cnb-require-jdk"
+version = "1.1.2"
+name = "Heroku Internal: Add CNB require for JDK"
+homepage = "https://github.com/heroku/buildpacks-jvm"
+description = "Internal buildpack that adds 'jdk' to the build plan."
```

Comment:
I'm not happy with the `name` and `description` here. If anyone has a better suggestion: let's hear it! :)